### PR TITLE
planner: update documents about cost model

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11825
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -966,15 +966,14 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
-- Default value: `1`
-- Value options: `1`, `2`
-- TiDB v6.2.0 introduces the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is more accurate than the previous version in internal tests.
-- To enable the Cost Model Version 2, you can set the `tidb_cost_model_version` to `2`. If you set this variable to `1`, the Cost Model Version 1 will be used.
+- Default value: `2`
+- Value options:
+    - `1`: enable the Cost Model Version 1.
+    - `2`: enable the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is introduced in TiDB v6.2.0 and is more accurate than the previous version in internal tests.
```
Comment (Oreoxmt) [inline system-variables.md:976]:
```suggestion
    - `2`: enable the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is introduced in TiDB v6.2.0 and is more accurate than the version 1 in internal tests.
```

File: cost-model.md [hunk 2]
```diff
@@ -40,11 +40,6 @@ The preceding example is simplified and only used to explain the basic principle
 
 ## Cost Model Version 2
 
-> **Warning:**
```
Comment (shichun-0415) [inline cost-model.md:43]: Add `Switching the version of the cost model might cause changes to query plans.` below Line 45.

File: cost-model.md [hunk 3]
```diff
@@ -40,11 +40,6 @@ The preceding example is simplified and only used to explain the basic principle
...
->
-> - Cost Model Version 2 is currently an experimental feature. It is not recommended that you use it for production environments.
-> - Switching the version of the cost model might cause changes to query plans.
-
 TiDB v6.2.0 introduces Cost Model Version 2, a new cost model.
 
 Cost Model Version 2 provides a more accurate regression calibration of the cost formula, adjusts some of the cost formulas, and is more accurate than the previous version of the cost formula.
```
Comment (Oreoxmt) [inline cost-model.md:45]:
```suggestion
Cost Model Version 2 provides a more accurate regression calibration of the cost formula, adjusts some of the cost formulas, and is more accurate than the previous version of the cost formula.

Switching the version of the cost model might cause changes to query plans.
```

File: cost-model.md [hunk 4]
```diff
@@ -40,13 +40,10 @@ The preceding example is simplified and only used to explain the basic principle
...
 
+Switching the version of the cost model might cause changes to query plans.
+
 To switch the version of cost model, you can set the [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-new-in-v620) variable.
```
Comment (Oreoxmt) [inline cost-model.md:49]:
```suggestion
To switch the version of cost model, you can set the [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-new-in-v620) variable. Switching the version of the cost model might cause changes to query plans.
```

File: system-variables.md [hunk 5]
```diff
@@ -999,16 +999,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Type: Integer
-- Default value: `1`
-- Range: `[1, 2]`
-- TiDB v6.2.0 introduces the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is more accurate than the previous version in internal tests.
-- To enable the Cost Model Version 2, you can set the `tidb_cost_model_version` to `2`. If you set this variable to `1`, the Cost Model Version 1 will be used.
+- Default value: `2`
+- Value options:
+    - `1`: enable the Cost Model Version 1.
```
Comment (Oreoxmt) [inline system-variables.md:1009]:
```suggestion
    - `1`: enable the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions. 
```

File: system-variables.md [hunk 6]
```diff
@@ -999,16 +999,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Type: Integer
-- Default value: `1`
-- Range: `[1, 2]`
-- TiDB v6.2.0 introduces the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is more accurate than the previous version in internal tests.
-- To enable the Cost Model Version 2, you can set the `tidb_cost_model_version` to `2`. If you set this variable to `1`, the Cost Model Version 1 will be used.
+- Default value: `2`
+- Value options:
+    - `1`: enable the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions.
```
Comment (TomShawn) [inline system-variables.md:1009]:
```suggestion
    - `1`: enables the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions. 
```

File: system-variables.md [hunk 7]
```diff
@@ -999,16 +999,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - Persists to cluster: Yes
 - Type: Integer
-- Default value: `1`
-- Range: `[1, 2]`
-- TiDB v6.2.0 introduces the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is more accurate than the previous version in internal tests.
-- To enable the Cost Model Version 2, you can set the `tidb_cost_model_version` to `2`. If you set this variable to `1`, the Cost Model Version 1 will be used.
+- Default value: `2`
+- Value options:
+    - `1`: enable the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions. 
+    - `2`: enable the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is introduced in TiDB v6.2.0 and is more accurate than the version 1 in internal tests.
```
Comment (TomShawn) [inline system-variables.md:1010]:
```suggestion
    - `2`: enables the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is introduced in TiDB v6.2.0 and is more accurate than the version 1 in internal tests.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: system-variables.md [hunk 8]
```diff
@@ -999,16 +999,16 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 > **Warning:**
```
Comment (Oreoxmt) [inline system-variables.md:1000]:
```suggestion
> **Note:**
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: system-variables.md [hunk 9]
```diff
@@ -997,18 +997,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - Persists to cluster: Yes
 - Type: Integer
-- Default value: `1`
-- Range: `[1, 2]`
-- TiDB v6.2.0 introduces the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is more accurate than the previous version in internal tests.
-- To enable the Cost Model Version 2, you can set the `tidb_cost_model_version` to `2`. If you set this variable to `1`, the Cost Model Version 1 will be used.
+- Default value: `2`
+- Value options:
+    - `1`: enables the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions.
+    - `2`: enables the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is introduced in TiDB v6.2.0 and is more accurate than the version 1 in internal tests.
```
Comment (Oreoxmt) [inline system-variables.md:1011]:
```suggestion
    - `1`: enables the Cost Model Version 1, which is used by default in TiDB v6.4.0 and earlier versions.
    - `2`: enables the [Cost Model Version 2](/cost-model.md#cost-model-version-2), which is generally available in TiDB v6.5.0 and is more accurate than the version 1 in internal tests.
```

Comment (qw4990) [reply system-variables.md:1011]: LGTM

**Review by qw4990 — APPROVED** [review]
LGTM

**Pull request merged**